packages/website/src/features/Packages/Interact.tsx

```diff
@@ -13,7 +13,11 @@ import {
   useBreakpointValue,
   useDisclosure,
 } from '@chakra-ui/react';
-import { ChainArtifacts, ContractData } from '@usecannon/builder/src';
+import {
+  ChainArtifacts,
+  ContractData,
+  parseVariant,
+} from '@usecannon/builder/src';
```

Comment:
Shouldn't be also imported from root?
```suggestion
import { ChainArtifacts, ContractData } from '@usecannon/builder/src';
import { parseVariant } from '@usecannon/builder';
```